hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1877,6 +1901,10 @@ func (r *HostedClusterReconciler) reconcileIgnitionServer(ctx context.Context, c
 									Name:      "serving-cert",
 									MountPath: "/var/run/secrets/ignition/serving-cert",
 								},
+								{
+									Name:      "cache",
+									MountPath: "/payload-cache",
```

Comment:
this is mounted here but I can't see where the ignition local provider target this path, `opts.WorkDir` seems to not be set so defaults to "" and then I see `workDir, err := ioutil.TempDir(p.CacheDir, "get-payload")`